.github/workflows/test-stellar.yaml

```
@@ -74,3 +89,117 @@ jobs:
 
       - name: Deploy AxelarGateway
         run: node stellar/deploy-contract deploy AxelarGateway --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy InterchainTokenService
+        run: node stellar/deploy-contract deploy InterchainTokenService --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy AxelarExample
+        run: node stellar/deploy-contract deploy AxelarExample --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy Upgrader
+        run: node stellar/deploy-contract deploy Upgrader --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy Multicall
+        run: node stellar/deploy-contract deploy Multicall --version ${{ steps.commit_hash.outputs.hash }}
+
+      ###### Command: Gateway ######
+
+      - name: Gateway Paused
+        run: node stellar/contract.js paused AxelarGateway
+
+      - name: Gateway Pause
+        run: node stellar/contract.js pause AxelarGateway
+
+      - name: Gateway Unpause
+        run: node stellar/contract.js unpause AxelarGateway
+
+      - name: Gateway Call Contract
+        run: node stellar/gateway.js call-contract ${{ env.destinationChain }} ${{ env.destinationAddress }} ${{ env.data }}
+
+      - name: Gateway Rotate Signers
+        run: node stellar/gateway.js rotate --new-nonce test --signers wallet
+
+      ###### Command: ITS ######
+
+      - name: ITS Paused
+        run: node stellar/contract.js paused InterchainTokenService
+
+      - name: ITS Pause
+        run: node stellar/contract.js pause InterchainTokenService
+
+      - name: ITS Unpause
+        run: node stellar/contract.js unpause InterchainTokenService
+
+      - name: Add Trusted Chain
+        run: node stellar/its.js add-trusted-chains ${{ env.destinationChain }}
+
+      - name: Remove Trusted Chain
+        run: node stellar/its.js remove-trusted-chains ${{ env.destinationChain }}
+
+      - name: Add All Trusted Chains
+        run: node stellar/its.js add-trusted-chains all
+
+      - name: Remove All Trusted Chains
+        run: node stellar/its.js remove-trusted-chains all
+
+      ###### Command: ITS Example ######
+
+      - name: Prepare ITS Example Parameters
+        run: |
+          echo "name=TEST" >> $GITHUB_ENV
+          echo "symbol=TEST" >> $GITHUB_ENV
+          echo "decimal=18" >> $GITHUB_ENV
+          echo "salt=0x1234" >> $GITHUB_ENV
+          echo "amount=1" >> $GITHUB_ENV
+
+          # Note that tokenId is hardcoded since it's derivation must always be the same
+          echo "tokenId=0x1829aae91cedf3603a9441688a099b4fcbdb6733b464027c69814b80ef420ada" >> $GITHUB_ENV
+
+      - name: Add Trusted Chains
+        run: node stellar/its.js add-trusted-chains ${{ env.destinationChain }} ${{ env.itsHubAddress }}
```

```suggestion
        run: node stellar/its.js add-trusted-chains ${{ env.destinationChain }}
```
why is its hub address being added (just need to trust the chain)

---

updated